db_stress_tool/db_stress_test_base.cc

```diff
@@ -3429,6 +3429,23 @@ void StressTest::Open(SharedState* shared, bool reopen) {
       options_.compression_manager = mgr;
       options_.compression = kZSTD;
       options_.bottommost_compression = kZSTD;
+    } else if (!strcasecmp(FLAGS_compression_manager.c_str(), "randommixed")) {
+      // Currently limited to ZSTD compression. Table property compression_name
+      // needs to set to zstd for now even when there can be more than one
+      // algorithm in the table under your compressor.
+      if (!ZSTD_Supported()) {
+        fprintf(
+            stderr,
```

Comment:
mostly duplicate with above if branch - can we combine into one if branch and differentiate mixed and random mixed inside the branch?